src/js/controller/controller.js

```diff
@@ -386,18 +423,28 @@ Object.assign(Controller.prototype, {
             }
         }
 
-        function _checkAutoStart() {
+        const _checkAutoStart = () => {
```

Comment:
LGTM and it works when I test it locally. However, this feature seems to only work when `autostart` is configured to `true`. Should this feature force autostart when the `jw_start` is included?

Reply:
Looks like I was testing on the older PR build of jw-commercial. LGTM!